treeherder/model/models.py

```diff
@@ -1394,3 +1394,21 @@ class Meta:
     def __str__(self):
         return "{0} {1}".format(
             self.text_log_error.id, self.classified_failure.id)
+
+
+class PushActivity(models.Model):
+    """
+    Keep track of events in Treeherder.  This can be used for an event with
+    """
+    id = models.BigAutoField(primary_key=True)
+    push = models.ForeignKey(Push, on_delete=models.CASCADE)
+    time = models.DateTimeField(db_index=True)
```

Comment:
What time?  last updated?  time of event?  What do the other tables use?

Reply:
I was thinking time of event, yes.